handlers/chat.js

```diff
@@ -279,7 +279,7 @@ var onOtherJoinedChannel = function onOtherJoinedChannel(message) {
     /* Someone joined a chat channel you're in. */
     var otherJoined = Dota2.schema.lookupType("CMsgDOTAOtherJoinedChatChannel").decode(message);
     var channel = this._getChannelById(otherJoined.channel_id);
-    this.Logger.debug(otherJoined.steam_id + " joined channel " + channel.channel_name);
+    this.Logger.debug(otherJoined.steam_id + " joined channel " + channel ? channel.channel_name : " unknown");
```

Comment:
This will throw the same error, you would need to use a template literal for it to work
```javascript
this.Logger.debug(`${otherJoined.steam_id} joined channel ${channel ? channel.channel_name : "unknown"}`);
```

Reply:
You're right. Either that or parentheses; I wouldn't want to use literals for consistency with the project.